.github/workflows/check.yml

```
@@ -0,0 +1,38 @@
+        with:
+          python-version: "3.12"
+      - name: Validate ai-rulez configuration
+        run: npx --yes ai-rulez@${INFRA_COPILOT_AI_RULEZ_VERSION} validate
+      - name: Verify committed plugin payloads
+        run: npx --yes ai-rulez@${INFRA_COPILOT_AI_RULEZ_VERSION} verify --plugin
+      - name: Validate links and host adapters
+        run: python3 scripts/validate.py
+      - name: Test repository validators
+        run: python3 -m unittest discover -s tests
+      - name: Smoke-test OpenCode installation
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:18: Double quote to prevent globbing and word splitting

(shellcheck)